CHANGELOG.md

```diff
@@ -80,6 +88,7 @@ A brief description of the categories of changes:
   and use the `pip` backend to do the downloading. This was only available for
   users setting {bzl:obj}`pip.parse.experimental_index_url`, but now users have
   more options whilst we continue to work on stabilizing the experimental feature.
+* (toolchain) The support for freethreaded Python toolchains is now available. 
```

Comment:
small typo
```suggestion
* (toolchain) Support for freethreaded Python toolchains is now available. 
```

Should this also mention how to activate it, or point to docs on how to do so?